tests/acceptance/features/bootstrap/CliContext.php

```diff
@@ -86,6 +86,20 @@ public function theAdministratorResetsThePasswordOfUserUsingTheCLI(string $user,
 		$this->featureContext->updateUserPassword($user, $password);
 	}
 
+	/**
+	 * @When the administrator checks the backup consistency using the CLI
+	 *
+	 * @return void
+	 */
+	public function theAdministratorChecksTheBackupConsistencyUsingTheCli():void {
+		$path = getenv('STORAGE_USERS_OCIS_ROOT') ? getenv('STORAGE_USERS_OCIS_ROOT') : "storage/users";
```

Comment:
`~` needs to be resolved to HOME dir. or we can do `getenc('HOME')`
```suggestion
		$path = getenv('STORAGE_USERS_OCIS_ROOT') ?? "~/.ocis/storage/users";
```

Reply:
move it to a method. we need it in other tests as well